.gitignore

```diff
@@ -31,6 +31,7 @@ cmake_install.cmake
 
 CTestTestfile.cmake
 src/camera_autogen/
+src/edgeDetection_autogen/
```

Comment:
change this and prev line to src/*_autogen/ to catch this for all future autogen